src/plugins/intel_gpu/src/graph/compilation_context.cpp

```diff
@@ -10,16 +10,19 @@ namespace cldnn {
 
 class CompilationContext : public ICompilationContext {
 public:
-    using compilation_queue_t = InferenceEngine::ThreadSafeQueue<ICompilationContext::Task>;
+    using data_type = std::pair<size_t, ICompilationContext::Task>;
+    using data_list_type = std::list<data_type>;
```

Comment:
Now we just use it as queue. Should we still  maintain this as list?

Reply:
Replaced to std::dequeu